src/lightning_utilities/core/imports.py

```diff
@@ -128,7 +128,7 @@ def _check_requirement(self) -> None:
         try:
             req = Requirement(self.requirement)
             pkg_version = Version(_version(req.name))
-            self.available = req.specifier.contains(pkg_version) and (
+            self.available = req.specifier.contains(pkg_version, prereleases=True) and (
```

Comment:
just thinking about making this as argument so a user can disable it.

Reply:
It was handled like this implicitly in the previous implementation, so for now I'm adding it as a fix. I don't think we need to expose it right now. If we see a need, we can always do it later.

Reply:
so at least to have it as class variable? but fine either way...

Reply:
If you are fine either way, I prefer to keep it like this for now. The class variable would not be ideal, since changing it on the class would affect all version checks in the library, which seems undesirable. If a configuration of this is needed, I think exposing an argument would still be better.

Reply:
:green_circle: